src/logic/main.js

```diff
@@ -51,11 +54,101 @@ function isMoveLegal() {
     return true;
 }
 
+
+// ++++++++++++++++++BOT ALGORITHM ++++++++++++++++++++++
+
 function playBotMove() {
     GameState.currentPlayer = PlayerEnum.Bot;
-    return playGameMove(GameState.BotPile.cards[0].id);
+    let leadingCard = GameState.leadingCard;
+    let selectedCard;
+    let actionState = GameState.actionState;
+    let bot = GameState[PileTypeEnum.BotPile];
```

Comment:
Please change variable name bot to a better name (botPile for example),
And you don't have to do GameState[PileTypeEnum.BotPile]
you can instead do: GameState.BotPile

Reply:
done